catalogue/webapp/components/Download/Download.js

```diff
@@ -86,7 +86,7 @@ function getFormatString(format) {
 type Work = Object;
 type Props = {|
   work: Work,
-  licenseInfo: LicenseData[],
+  licenseInfo: LicenseUIData[],
```

Comment:
I thought we'd been trying to avoid `IU` type naming as it caused confusion as it's all UI?

Reply:
This was to distinguish between the license data we get from the API and the license data we end up with, but you're right it's not really needed.